assets/templates/signup.html

```diff
@@ -110,9 +110,9 @@ <h5>email, automation & data marketing</h5>
                 <label for="password">Contraseña: <a toggle="#password-field" class="ms-icon icon-view show-hide"> <span class="content-eye">Mostrar</a></label>
                 <input type="password" name="password" id="password-field" placeholder="Escribe tu clave secreta">
                 <div class="wrapper-password">
-                  <p class="password-message"><span class="waiting-message">8 caracteres como mínimo</span><span class="waiting-message">Un dígito</span></p>
-                  <!-- <p class="password-message"><span class="lack-message">8 caracteres como mínimo</span><span class="lack-message">Un dígito</span></p>
-                  <p class="password-message"><span class="complete-message">8 caracteres como mínimo</span><span class="complete-message">Un dígito</span></p>
+                  <p class="password-message"><span class="waiting-message">8 caracteres como mínimo</span><span class="waiting-message">Una letra</span><span class="waiting-message">Un número</span></p>
+                  <!-- <p class="password-message"><span class="lack-message">8 caracteres como mínimo</span><span class="lack-message">Una letra</span><span class="lack-message">Un número</span></p>
+                  <p class="password-message"><span class="complete-message">8 caracteres como mínimo</span><span class="complete-message">Una letra</span><span class="complete-message">Un número</span></p>
```

Comment:
Does it require changes in webapp?

Reply:
Ok, I do not know where those messages are, but I can add them if you tell me where

Reply:
@cbernat has a related PR open, if it requires changes in webapp markup, please let us know.

Reply:
@cbernat implements it!

Reply:
Done in https://github.com/FromDoppler/doppler-webapp/pull/126